src/utilities.jl

```diff
@@ -119,6 +119,37 @@ function split_vertices_set!(
     return (active_set, right_as)
 end
 
+function dicg_split_vertices_set!(active_set::FrankWolfe.ActiveSet{T,R}, tree, vidx::Int, ::IntegerBounds;kwargs...)where {T,R}
+    blmo = tree.root.problem.tlmo.blmo
+    x = FrankWolfe.get_active_set_iterate(active_set)
+    n = length(x)
+    x0_left = copy(x)
+    x0_right = copy(x)
+    if typeof(blmo).name.wrapper == ManagedBoundedLMO
```

Comment:
You don't need the case distinguision here. In any case, it is 
x_left[idx] = floor(x[idx])
and 
x_right[idx] = ceil(x[idx])